apperals-frontend/src/components/navigation/ButtonRow.js

```diff
@@ -1,12 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './ButtonRow.css';
 /*import fashionImage from '../assets/A_black_background.jpg';*/
 import { Link } from 'react-router-dom';
 
 function ButtonRow() {
```

Comment:
Rename Function to Navigation or NavigationBar.
eg :  export const Navigation = () => {
    return (
        ....
    )
}

this would automatically export the component. Dont need to export the component at the end of the file.
PS: Could learn more on arrow function in JS